CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+cmd/src/lsif* @sourcegraph/team-graph
```

Comment:
I just realized -- should we put this file within the `cmd/src` directory? I think it's common to nest CODEOWNERS in large-ish codebases like ours.